### ISSUE TITLE
docker: docker-compose up fails in copying step "no source files were specified"

### ISSUE THREAD

**Issue by schicho** [open-meteo/open-meteo#229]
When trying to run the image, the process stops in 17/21

```sh
Step 17/21 : COPY --from=build --chown=vapor:vapor /build/.build/release/*.resources /app/Resources/
COPY failed: no source files were specified
```

I have attempted to find a solution, but was unsuccessful.
My guesses are:

- wildcards
- no files match the wildcard and subsequently fails (could not test or verify this)
- build dir is in the .dockerignore

However, in the previous copy steps, the copying from the build dir works fine.

```sh
Step 15/21 : COPY --from=build --chown=vapor:vapor /build/.build/release /app
 ---> 4e2fc8e15c58
Step 16/21 : COPY --from=build --chown=vapor:vapor /build/Resources /app/Resources
 ---> 454ced06b2a1
Step 17/21 : COPY --from=build --chown=vapor:vapor /build/.build/release/*.resources /app/Resources/
COPY failed: no source files were specified
ERROR: Service 'open-meteo' failed to build : Build failed
```

I suspect guess number 2 to be the issue.
There are only work-arounds for it like mentioned in https://stackoverflow.com/a/46801962

I don't know which solution is preferable.

**Comment by schicho** [comment]
I am using Linux (Ubuntu 20.04)

I have extended the Dockerfile as follows



```Dockerfile
...
ENV DEBIAN_FRONTEND=noninteractive
RUN apt update && apt install -y libnetcdf-dev libeccodes-dev libbz2-dev build-essential && rm -rf /var/lib/apt/lists/*

# Compile with optimizations
RUN swift build --enable-test-discovery -c release

# Show if files in .build/release
RUN pwd
RUN ls -la
RUN ls -la .build
RUN ls -la .build/release/
RUN ls -la .build/release/*.resources
...
```

I get the following output:

```shell
Step 9/26 : RUN pwd
 ---> Running in d2b709d10e6b
/build
Removing intermediate container d2b709d10e6b
 ---> 1943cc4154c9
Step 10/26 : RUN ls -la
 ---> Running in 99584ae6284c
total 140
drwxr-xr-x 1 root root  4096 Feb 20 13:51 .
drwxr-xr-x 1 root root  4096 Feb 20 13:57 ..
drwxr-xr-x 1 root root  4096 Feb 20 13:57 .build
-rw-rw-r-- 1 root root    20 Feb  6 17:40 .codebeatignore
-rw-rw-r-- 1 root root    18 Feb  6 17:40 .dockerignore
drwxrwxr-x 8 root root  4096 Feb 20 13:23 .git
drwxrwxr-x 3 root root  4096 Feb  6 17:40 .github
-rw-rw-r-- 1 root root    97 Feb  6 17:40 .gitignore
-rw-rw-r-- 1 root root  1838 Feb 20 13:51 Dockerfile
-rw-rw-r-- 1 root root   363 Feb  6 17:40 Dockerfile.development
-rw-rw-r-- 1 root root 34522 Feb  6 17:40 LICENSE
-rw-rw-r-- 1 root root  7137 Feb  6 17:40 Package.resolved
-rw-rw-r-- 1 root root  2494 Feb  6 17:40 Package.swift
drwxrwxr-x 7 root root  4096 Feb  6 17:40 Public
-rw-rw-r-- 1 root root  7213 Feb  6 17:40 README.md
drwxrwxr-x 3 root root  4096 Feb  6 17:40 Resources
drwxrwxr-x 9 root root  4096 Feb  6 17:40 Sources
drwxrwxr-x 3 root root  4096 Feb  6 17:40 Tests
drwxrwxr-x 2 root root  4096 Feb  6 17:40 build
-rw-rw-r-- 1 root root   922 Feb  6 17:40 docker-compose.yml
drwxrwxr-x 2 root root  4096 Feb  6 17:40 docs
-rw-rw-r-- 1 root root  7454 Feb  6 17:40 openapi.yml
-rw-rw-r-- 1 root root  7573 Feb  6 17:40 openapi_historical_weather_api.yml
Removing intermediate container 99584ae6284c
 ---> 20d516198324
Step 11/26 : RUN ls -la .build
 ---> Running in 50e416bc5bef
total 2464
drwxr-xr-x  1 root root    4096 Feb 20 13:57 .
drwxr-xr-x  1 root root    4096 Feb 20 13:51 ..
drwxr-xr-x  2 root root    4096 Feb 20 13:23 artifacts
drwxr-xr-x 28 root root    4096 Feb 20 13:24 checkouts
lrwxrwxrwx  1 root root      32 Feb 20 13:57 release -> x86_64-unknown-linux-gnu/release
-rw-r--r--  1 root root 2479018 Feb 20 13:52 release.yaml
drwxr-xr-x 28 root root    4096 Feb 20 13:24 repositories
-rw-r--r--  1 root root   13631 Feb 20 13:51 workspace-state.json
drwxr-xr-x  3 root root    4096 Feb 20 13:57 x86_64-unknown-linux-gnu
Removing intermediate container 50e416bc5bef
 ---> 5288905784b6
Step 12/26 : RUN ls -la .build/release/
 ---> Running in bdd2b1c7133d
total 108292
drwxr-xr-x 73 root root    12288 Feb 20 13:57 .
drwxr-xr-x  3 root root     4096 Feb 20 13:57 ..
drwxr-xr-x  2 root root     4096 Feb 20 13:57 Algorithms.build
-rw-r--r--  1 root root   138356 Feb 20 13:52 Algorithms.swiftdoc
-rw-r--r--  1 root root  1178012 Feb 20 13:52 Algorithms.swiftmodule
-rw-r--r--  1 root root   243816 Feb 20 13:52 Algorithms.swiftsourceinfo
drwxr-xr-x 21 root root     4096 Feb 20 13:57 App.build
-rw-r--r--  1 root root    14004 Feb 20 13:56 App.swiftdoc
-rw-r--r--  1 root root   903784 Feb 20 13:56 App.swiftmodule
-rw-r--r--  1 root root    55548 Feb 20 13:56 App.swiftsourceinfo
drwxr-xr-x  5 root root     4096 Feb 20 13:55 AsyncHTTPClient.build
-rw-r--r--  1 root root    83976 Feb 20 13:55 AsyncHTTPClient.swiftdoc
-rw-r--r--  1 root root   582776 Feb 20 13:55 AsyncHTTPClient.swiftmodule
-rw-r--r--  1 root root   116640 Feb 20 13:55 AsyncHTTPClient.swiftsourceinfo
drwxr-xr-x  5 root root     4096 Feb 20 13:55 AsyncKit.build
-rw-r--r--  1 root root    83536 Feb 20 13:54 AsyncKit.swiftdoc
-rw-r--r--  1 root root  1347272 Feb 20 13:54 AsyncKit.swiftmodule
-rw-r--r--  1 root root    69584 Feb 20 13:54 AsyncKit.swiftsourceinfo
drwxr-xr-x  3 root root     4096 Feb 20 13:52 Atomics.build
-rw-r--r--  1 root root    80184 Feb 20 13:52 Atomics.swiftdoc
-rw-r--r--  1 root root  1356640 Feb 20 13:52 Atomics.swiftmodule
-rw-r--r--  1 root root   158896 Feb 20 13:52 Atomics.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:54 Backtrace.build
-rw-r--r--  1 root root      828 Feb 20 13:53 Backtrace.swiftdoc
-rw-r--r--  1 root root    17868 Feb 20 13:53 Backtrace.swiftmodule
-rw-r--r--  1 root root     1668 Feb 20 13:53 Backtrace.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:53 CAsyncHTTPClient.build
drwxr-xr-x  2 root root     4096 Feb 20 13:53 CBacktrace.build
drwxr-xr-x  3 root root     4096 Feb 20 13:53 CCryptoBoringSSL.build
drwxr-xr-x  2 root root     4096 Feb 20 13:53 CCryptoBoringSSLShims.build
drwxr-xr-x  3 root root     4096 Feb 20 13:53 CHelper.build
drwxr-xr-x  3 root root     4096 Feb 20 13:53 CNIOAtomics.build
drwxr-xr-x  4 root root     4096 Feb 20 13:53 CNIOBoringSSL.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CNIOBoringSSLShims.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CNIODarwin.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CNIOExtrasZlib.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CNIOLLHTTP.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CNIOLinux.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CNIOSHA1.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CNIOWindows.build
drwxr-xr-x  3 root root     4096 Feb 20 13:52 CTurboPFor.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CVaporBcrypt.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CVaporURLParser.build
drwxr-xr-x  3 root root     4096 Feb 20 13:52 CZoneDetect.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 Collections.build
-rw-r--r--  1 root root      376 Feb 20 13:52 Collections.swiftdoc
-rw-r--r--  1 root root    13720 Feb 20 13:52 Collections.swiftmodule
-rw-r--r--  1 root root      632 Feb 20 13:52 Collections.swiftsourceinfo
drwxr-xr-x  9 root root     4096 Feb 20 13:52 ConsoleKit.build
-rw-r--r--  1 root root    79124 Feb 20 13:52 ConsoleKit.swiftdoc
-rw-r--r--  1 root root   256464 Feb 20 13:52 ConsoleKit.swiftmodule
-rw-r--r--  1 root root   104652 Feb 20 13:52 ConsoleKit.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 CoreMetrics.build
-rw-r--r--  1 root root    33116 Feb 20 13:52 CoreMetrics.swiftdoc
-rw-r--r--  1 root root   133132 Feb 20 13:52 CoreMetrics.swiftmodule
-rw-r--r--  1 root root    34844 Feb 20 13:52 CoreMetrics.swiftsourceinfo
drwxr-xr-x 14 root root     4096 Feb 20 13:54 Crypto.build
-rw-r--r--  1 root root    51944 Feb 20 13:54 Crypto.swiftdoc
-rw-r--r--  1 root root   574004 Feb 20 13:54 Crypto.swiftmodule
-rw-r--r--  1 root root   173192 Feb 20 13:54 Crypto.swiftsourceinfo
drwxr-xr-x  3 root root     4096 Feb 20 13:53 CryptoBoringWrapper.build
-rw-r--r--  1 root root     4864 Feb 20 13:53 CryptoBoringWrapper.swiftdoc
-rw-r--r--  1 root root    50408 Feb 20 13:53 CryptoBoringWrapper.swiftmodule
-rw-r--r--  1 root root     7060 Feb 20 13:53 CryptoBoringWrapper.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 DequeModule.build
-rw-r--r--  1 root root    45524 Feb 20 13:52 DequeModule.swiftdoc
-rw-r--r--  1 root root   315304 Feb 20 13:52 DequeModule.swiftmodule
-rw-r--r--  1 root root    70908 Feb 20 13:52 DequeModule.swiftsourceinfo
drwxr-xr-x  6 root root     4096 Feb 20 13:54 IkigaJSON.build
-rw-r--r--  1 root root    20044 Feb 20 13:54 IkigaJSON.swiftdoc
-rw-r--r--  1 root root   209788 Feb 20 13:54 IkigaJSON.swiftmodule
-rw-r--r--  1 root root    41124 Feb 20 13:54 IkigaJSON.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:56 Leaf.build
-rw-r--r--  1 root root     3224 Feb 20 13:56 Leaf.swiftdoc
-rw-r--r--  1 root root    70996 Feb 20 13:56 Leaf.swiftmodule
-rw-r--r--  1 root root    10896 Feb 20 13:56 Leaf.swiftsourceinfo
drwxr-xr-x  9 root root     4096 Feb 20 13:55 LeafKit.build
-rw-r--r--  1 root root    33544 Feb 20 13:54 LeafKit.swiftdoc
-rw-r--r--  1 root root   272440 Feb 20 13:54 LeafKit.swiftmodule
-rw-r--r--  1 root root    68368 Feb 20 13:54 LeafKit.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 Logging.build
-rw-r--r--  1 root root    26692 Feb 20 13:52 Logging.swiftdoc
-rw-r--r--  1 root root   167304 Feb 20 13:52 Logging.swiftmodule
-rw-r--r--  1 root root    34144 Feb 20 13:52 Logging.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 Metrics.build
-rw-r--r--  1 root root     1952 Feb 20 13:52 Metrics.swiftdoc
-rw-r--r--  1 root root    39060 Feb 20 13:52 Metrics.swiftmodule
-rw-r--r--  1 root root     2328 Feb 20 13:52 Metrics.swiftsourceinfo
drwxr-x---  9 root root     4096 Feb 20 13:53 ModuleCache
drwxr-xr-x  5 root root     4096 Feb 20 13:55 MultipartKit.build
-rw-r--r--  1 root root    11812 Feb 20 13:54 MultipartKit.swiftdoc
-rw-r--r--  1 root root   118364 Feb 20 13:54 MultipartKit.swiftmodule
-rw-r--r--  1 root root    23724 Feb 20 13:54 MultipartKit.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:54 NIO.build
-rw-r--r--  1 root root      368 Feb 20 13:54 NIO.swiftdoc
-rw-r--r--  1 root root    13712 Feb 20 13:54 NIO.swiftmodule
-rw-r--r--  1 root root      604 Feb 20 13:54 NIO.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:53 NIOConcurrencyHelpers.build
-rw-r--r--  1 root root    29472 Feb 20 13:53 NIOConcurrencyHelpers.swiftdoc
-rw-r--r--  1 root root   295956 Feb 20 13:53 NIOConcurrencyHelpers.swiftmodule
-rw-r--r--  1 root root    71952 Feb 20 13:53 NIOConcurrencyHelpers.swiftsourceinfo
drwxr-xr-x  3 root root     4096 Feb 20 13:54 NIOCore.build
-rw-r--r--  1 root root   521632 Feb 20 13:53 NIOCore.swiftdoc
-rw-r--r--  1 root root  2155632 Feb 20 13:53 NIOCore.swiftmodule
-rw-r--r--  1 root root   536072 Feb 20 13:53 NIOCore.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:54 NIOEmbedded.build
-rw-r--r--  1 root root    55640 Feb 20 13:54 NIOEmbedded.swiftdoc
-rw-r--r--  1 root root   202676 Feb 20 13:54 NIOEmbedded.swiftmodule
-rw-r--r--  1 root root    51468 Feb 20 13:54 NIOEmbedded.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:54 NIOExtras.build
-rw-r--r--  1 root root    57116 Feb 20 13:54 NIOExtras.swiftdoc
-rw-r--r--  1 root root   242676 Feb 20 13:54 NIOExtras.swiftmodule
-rw-r--r--  1 root root    57168 Feb 20 13:54 NIOExtras.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:55 NIOFoundationCompat.build
-rw-r--r--  1 root root    19540 Feb 20 13:55 NIOFoundationCompat.swiftdoc
-rw-r--r--  1 root root    86608 Feb 20 13:55 NIOFoundationCompat.swiftmodule
-rw-r--r--  1 root root    16448 Feb 20 13:55 NIOFoundationCompat.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:55 NIOHPACK.build
-rw-r--r--  1 root root    38096 Feb 20 13:54 NIOHPACK.swiftdoc
-rw-r--r--  1 root root   522752 Feb 20 13:54 NIOHPACK.swiftmodule
-rw-r--r--  1 root root    47128 Feb 20 13:54 NIOHPACK.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:54 NIOHTTP1.build
-rw-r--r--  1 root root    57316 Feb 20 13:54 NIOHTTP1.swiftdoc
-rw-r--r--  1 root root   403392 Feb 20 13:54 NIOHTTP1.swiftmodule
-rw-r--r--  1 root root    98676 Feb 20 13:54 NIOHTTP1.swiftsourceinfo
drwxr-xr-x  4 root root     4096 Feb 20 13:55 NIOHTTP2.build
-rw-r--r--  1 root root   101392 Feb 20 13:55 NIOHTTP2.swiftdoc
-rw-r--r--  1 root root   769312 Feb 20 13:55 NIOHTTP2.swiftmodule
-rw-r--r--  1 root root   144460 Feb 20 13:55 NIOHTTP2.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:54 NIOHTTPCompression.build
-rw-r--r--  1 root root    13960 Feb 20 13:54 NIOHTTPCompression.swiftdoc
-rw-r--r--  1 root root   145348 Feb 20 13:54 NIOHTTPCompression.swiftmodule
-rw-r--r--  1 root root    17632 Feb 20 13:54 NIOHTTPCompression.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:54 NIOPosix.build
-rw-r--r--  1 root root    84108 Feb 20 13:54 NIOPosix.swiftdoc
-rw-r--r--  1 root root   475216 Feb 20 13:54 NIOPosix.swiftmodule
-rw-r--r--  1 root root    83460 Feb 20 13:54 NIOPosix.swiftsourceinfo
drwxr-xr-x  5 root root     4096 Feb 20 13:54 NIOSOCKS.build
-rw-r--r--  1 root root    18888 Feb 20 13:54 NIOSOCKS.swiftdoc
-rw-r--r--  1 root root   172956 Feb 20 13:54 NIOSOCKS.swiftmodule
-rw-r--r--  1 root root    26204 Feb 20 13:54 NIOSOCKS.swiftsourceinfo
drwxr-xr-x  3 root root     4096 Feb 20 13:54 NIOSSL.build
-rw-r--r--  1 root root    78120 Feb 20 13:54 NIOSSL.swiftdoc
-rw-r--r--  1 root root   599136 Feb 20 13:54 NIOSSL.swiftmodule
-rw-r--r--  1 root root   113616 Feb 20 13:54 NIOSSL.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 NIOSSLHTTP1Client.product
drwxr-xr-x  2 root root     4096 Feb 20 13:54 NIOTLS.build
-rw-r--r--  1 root root     7108 Feb 20 13:54 NIOTLS.swiftdoc
-rw-r--r--  1 root root    59332 Feb 20 13:54 NIOTLS.swiftmodule
-rw-r--r--  1 root root     8120 Feb 20 13:54 NIOTLS.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 NIOTLSServer.product
drwxr-xr-x  2 root root     4096 Feb 20 13:55 NIOTransportServices.build
-rw-r--r--  1 root root      992 Feb 20 13:55 NIOTransportServices.swiftdoc
-rw-r--r--  1 root root    40492 Feb 20 13:55 NIOTransportServices.swiftmodule
-rw-r--r--  1 root root     6408 Feb 20 13:55 NIOTransportServices.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:54 NIOWebSocket.build
-rw-r--r--  1 root root    36428 Feb 20 13:54 NIOWebSocket.swiftdoc
-rw-r--r--  1 root root   206548 Feb 20 13:54 NIOWebSocket.swiftmodule
-rw-r--r--  1 root root    43948 Feb 20 13:54 NIOWebSocket.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 OpenMeteoApiPackageTests.product
drwxr-xr-x  6 root root     4096 Feb 20 13:52 OrderedCollections.build
-rw-r--r--  1 root root   303520 Feb 20 13:52 OrderedCollections.swiftdoc
-rw-r--r--  1 root root   755624 Feb 20 13:52 OrderedCollections.swiftmodule
-rw-r--r--  1 root root   315760 Feb 20 13:52 OrderedCollections.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 RealModule.build
-rw-r--r--  1 root root    37988 Feb 20 13:52 RealModule.swiftdoc
-rw-r--r--  1 root root   157080 Feb 20 13:52 RealModule.swiftmodule
-rw-r--r--  1 root root    53996 Feb 20 13:52 RealModule.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 RoutingKit.build
-rw-r--r--  1 root root    11084 Feb 20 13:52 RoutingKit.swiftdoc
-rw-r--r--  1 root root    46380 Feb 20 13:52 RoutingKit.swiftmodule
-rw-r--r--  1 root root    13060 Feb 20 13:52 RoutingKit.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 SwiftEccodes.build
-rw-r--r--  1 root root     5492 Feb 20 13:52 SwiftEccodes.swiftdoc
-rw-r--r--  1 root root    39904 Feb 20 13:52 SwiftEccodes.swiftmodule
-rw-r--r--  1 root root     7060 Feb 20 13:52 SwiftEccodes.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 SwiftNetCDF.build
-rw-r--r--  1 root root    40764 Feb 20 13:52 SwiftNetCDF.swiftdoc
-rw-r--r--  1 root root   131468 Feb 20 13:52 SwiftNetCDF.swiftmodule
-rw-r--r--  1 root root    56136 Feb 20 13:52 SwiftNetCDF.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:56 SwiftPFor2D.build
-rw-r--r--  1 root root    13424 Feb 20 13:56 SwiftPFor2D.swiftdoc
-rw-r--r--  1 root root    73800 Feb 20 13:56 SwiftPFor2D.swiftmodule
-rw-r--r--  1 root root    24952 Feb 20 13:56 SwiftPFor2D.swiftsourceinfo
drwxr-xr-x  3 root root     4096 Feb 20 13:52 SwiftTimeZoneLookup.build
-rw-r--r--  1 root root     1820 Feb 20 13:52 SwiftTimeZoneLookup.swiftdoc
-rw-r--r--  1 root root    24840 Feb 20 13:52 SwiftTimeZoneLookup.swiftmodule
-rw-r--r--  1 root root     3288 Feb 20 13:52 SwiftTimeZoneLookup.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 SwiftTimeZoneLookup_SwiftTimeZoneLookup.resources
drwxr-xr-x 29 root root     4096 Feb 20 13:56 Vapor.build
-rw-r--r--  1 root root   281816 Feb 20 13:56 Vapor.swiftdoc
-rw-r--r--  1 root root  1536216 Feb 20 13:56 Vapor.swiftmodule
-rw-r--r--  1 root root   543620 Feb 20 13:56 Vapor.swiftsourceinfo
drwxr-xr-x  3 root root     4096 Feb 20 13:55 WebSocketKit.build
-rw-r--r--  1 root root     1316 Feb 20 13:55 WebSocketKit.swiftdoc
-rw-r--r--  1 root root    63152 Feb 20 13:55 WebSocketKit.swiftmodule
-rw-r--r--  1 root root    13136 Feb 20 13:55 WebSocketKit.swiftsourceinfo
drwxr-xr-x  3 root root     4096 Feb 20 13:52 _AtomicsShims.build
drwxr-xr-x  2 root root     4096 Feb 20 13:52 _NIODataStructures.build
-rw-r--r--  1 root root      384 Feb 20 13:52 _NIODataStructures.swiftdoc
-rw-r--r--  1 root root   109980 Feb 20 13:52 _NIODataStructures.swiftmodule
-rw-r--r--  1 root root    11084 Feb 20 13:52 _NIODataStructures.swiftsourceinfo
drwxr-xr-x  2 root root     4096 Feb 20 13:52 _NumericsShims.build
-rw-r--r--  1 root root   996462 Feb 20 13:52 description.json
-rwxr-xr-x  1 root root 86210800 Feb 20 13:57 openmeteo-api
drwxr-xr-x  2 root root     4096 Feb 20 13:52 openmeteo-api.product
drwxr-xr-x  2 root root     4096 Feb 20 13:57 openmeteo_api.build
-rw-r--r--  1 root root      380 Feb 20 13:57 openmeteo_api.swiftdoc
-rw-r--r--  1 root root    18028 Feb 20 13:57 openmeteo_api.swiftmodule
-rw-r--r--  1 root root      596 Feb 20 13:57 openmeteo_api.swiftsourceinfo
Removing intermediate container bdd2b1c7133d
 ---> fb3253839fd9
Step 13/26 : RUN ls -la .build/release/*.resources
 ---> Running in 646d0c8fcb1e
total 5496
drwxr-xr-x  2 root root    4096 Feb 20 13:52 .
drwxr-xr-x 73 root root   12288 Feb 20 13:57 ..
-r--r--r--  1 root root     124 Feb 20 13:52 README.md
-r--r--r--  1 root root 1098340 Feb 20 13:52 timezone16.bin
-r--r--r--  1 root root 4505306 Feb 20 13:52 timezone21.bin
Removing intermediate container 646d0c8fcb1e
 ---> 21b89a50d230
```


When you CRTL+F the output, you can find the file. I am somewhat confused about all this now.

**Comment by schicho** [comment]
- maybe it is the .dockerignore file after all?
- the copying over symbolic links is broken?

**Comment by patrick-zippenfenig** [comment]
Can you replace 
```COPY --from=build --chown=vapor:vapor /build/.build/release/*.resources /app/Resources/```
 by 
```COPY --from=build --chown=vapor:vapor /build/.build/release/SwiftTimeZoneLookup_SwiftTimeZoneLookup.resource /app/Resources/```
and try again?

**Comment by schicho** [comment]
Does not exist, magically?

```
Step 17/21 : COPY --from=build --chown=vapor:vapor /build/.build/release/SwiftTimeZoneLookup_SwiftTimeZoneLookup.resource /app/Resources/
COPY failed: stat build/.build/release/SwiftTimeZoneLookup_SwiftTimeZoneLookup.resource: file does not exist
```

**Comment by schicho** [comment]
> Does not exist, magically?
> 
> ```
> Step 17/21 : COPY --from=build --chown=vapor:vapor /build/.build/release/SwiftTimeZoneLookup_SwiftTimeZoneLookup.resource /app/Resources/
> COPY failed: stat build/.build/release/SwiftTimeZoneLookup_SwiftTimeZoneLookup.resource: file does not exist
> ```
> 
> ahh well, let me try again. I just copied your answer and it misses the final 's' in resources.

This fixed it!

just need to avoid the typo ;)

```Dockerfile
COPY --from=build --chown=vapor:vapor /build/.build/release/SwiftTimeZoneLookup_SwiftTimeZoneLookup.resources /app/Resources/
```

**Comment by patrick-zippenfenig** [comment]
Ah nice. Can you create a PR? I do not expect to have more resources files in the near future

**Comment by schicho** [comment]
PR #230

**Comment by j2l** [comment]
Before writing a ticket, I'm also on Ubuntu 22.04 and I can't run the image:
```
#8 7.423 W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/jammy/InRelease  Temporary failure resolving 'archive.ubuntu.com'
#8 7.423 W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/jammy-updates/InRelease  Temporary failure resolving 'archive.ubuntu.com'
#8 7.423 W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/jammy-backports/InRelease  Temporary failure resolving 'archive.ubuntu.com'
#8 7.423 W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/jammy-security/InRelease  Temporary failure resolving 'security.ubuntu.com'
```
AFTER starting to download: 
` => => sha256:97a89c39ae29f9b99db84d92734425a9a32030ef3065b71fa83b745433f94f3e 54.53MB / 103.47MB                                                  7.5s`

Did you have something similar @schicho ?
Do you know a fix ti this behavior guys?
Thanks

**Comment by schicho** [comment]
Seems like a network issue. Never had that occur and probably unrelated to this.
